src/openforms/config/models.py

```diff
@@ -649,6 +656,33 @@ def as_dict(self):
         return {k: list(v) for k, v in ret.items()}
 
 
+class CSPSettingManager(models.Manager.from_queryset(CSPSettingQuerySet)):
+    def set_for(self, obj: models.Model, settings: list[Tuple[str, str]]) -> None:
```

Comment:
```suggestion
    def set_for(self, obj: models.Model, settings: list[tuple[str, str]]) -> None:
```

No need to import `from typing import Tuple` on Python 3.10